clients/fides-js/src/fides-ext-gpp.ts

```diff
@@ -224,6 +230,7 @@ export const initializeGppCmpApi = () => {
         cmpApi,
         cookie: event.detail,
         experience: window.Fides.experience,
+        forcedGpp: window.Fides.options.forceGpp,
```

Comment:
can we be consistent with naming, e.g. making every reference either `forcedGpp` or `forceGpp`?

Reply:
🤔 one is more like `isForced` where we're checking for the property and the other is more like `doForce` where we're making the assignment so the naming difference was intentional like that, but I can see how that's maybe a little confusing.